README.md

```diff
@@ -1 +1,69 @@
-# microverse
\ No newline at end of file
+# Hello Microverse
+
+> Hello Microverse is first project for microverse .
+
+## Built With
```

Comment:
- [ ] I noticed that you are still missing the Table of contents for the README file.
- [ ]  Just as the first reviewer suggested, Kindly add the Table of Contents as it is essential for navigation through your README file :100: 
- [ OPTIONAL ] Please note that all the sections of the [Microverse Readme template](https://github.com/microverseinc/readme-template) are compulsory with the exception of the (`FAQs` section). So I will suggest you include all the other sections you exempted from the README template and fill them with the right information even though this is an optional comment :100: 
- In this case, the section you are missing is the `Future Features` section

Reply:
Thank you for finding out my mistakes, I will make the required changes.